src/Controller/EnMarche/AssessorManagerController.php

```diff
@@ -0,0 +1,236 @@
+        if (!$assessorRequestRepository->isManagedBy($this->getUser(), $assessorRequest)) {
+            throw $this->createNotFoundException(sprintf('User is not allowed to managed the assessor request with id %d.', $assessorRequest->getId()));
+        }
+
+        if ($assessorRequest->isDisabled() || !\in_array($votePlace, $manager->getMatchingVotePlaces($assessorRequest))) {
+            throw $this->createNotFoundException('No vote place for this request.');
+        }
+
+        $form = $this->createForm(FormType::class)
+            ->handleRequest($request)
+        ;
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $manager->processAssessorRequest($assessorRequest, $votePlace, true);
+            $this->addFlash('info', 'assessor.associate.success');
+
+            return $this->redirectToRoute('app_assessor_manager_request', ['id' => $assessorRequest->getId()]);
+        }
+
+        return $this->render('assessor_manager/associate.html.twig', [
+            'form' => $form->createView(),
+            'request' => $assessorRequest,
+            'votePlace' => $votePlace,
+        ]);
+    }
+
+    /**
+     * @Route(
+     *     "/demande/{id}/desassocier",
+     *     requirements={"id": "\d+"},
+     *     name="app_assessor_manager_request_deassociate"
+     * )
+     * @Method("GET|POST")
+     */
+    public function assessorRequestDessociateAction(
+        Request $request,
+        AssessorRequest $assessorRequest,
+        AssessorRequestRepository $repository,
+        AssessorManager $manager
+    ): Response {
+        if (!$assessorRequest->getVoteCity()) {
+            throw $this->createNotFoundException('This assessor request has no vote place.');
+        }
+
+        if (!$repository->isManagedBy($this->getUser(), $assessorRequest)) {
+            throw $this->createNotFoundException('Assessor request is not managed by the current user.');
+        }
+
+        $form = $this->createForm(FormType::class);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $manager->unprocessProcurationRequest($assessorRequest);
+            $this->addFlash('info', 'assessor.deassociate.success');
+
+            return $this->redirectToRoute('app_assessor_manager_request', ['id' => $assessorRequest->getId()]);
+        }
+
+        return $this->render('assessor_manager/deassociate.html.twig', [
+            'form' => $form->createView(),
+            'request' => $assessorRequest,
+            'votePlace' => $assessorRequest->getVotePlace(),
+        ]);
+    }
+
+    /**
+     * @Route(
+     *     "/vote-places/{id}/{action}",
+     *     requirements={ "id": "\d+", "action": AppBundle\Entity\ActionEnum::ACTIONS_URI_REGEX },
+     *     name="app_assessor_manager_request_transform"
+     * )
+     * @Method("GET")
+     */
+    public function assessorRequestTransformAction(int $id, string $action, AssessorManager $manager): Response
```

Comment:
this action seems disable/enable an assessor request and not a vote place, why do we find `/vote-places/` in the URI?

Reply:
Indeed, not used yet for vote places, maybe later for manual attribution.